platform/linux-dpdk/include/odp/api/plat/version_types.h

```diff
@@ -0,0 +1 @@
+../../../../../linux-generic/include/odp/api/plat/version_types.h
```

Comment:
These latest comments show why a merge has to be more carefully considered. Today, the makefiles pretty much assume that platform contains only a single platform. Obviously that needs to change if we're supporting multiple platforms/modules in a single repo. So the answer isn't so much to modify those in platform/linux-dpdk as to factor them out into a more global build system that recognizes that the repo now consists of multiple platforms that can be selected by new ./configure options. That restructure work can just as easily be done from the odp-dpdk-import branch as part of staging it into cloud-dev and doing it that way will avoid a lot of "backing out" commits that are undoing stuff we don't want anyway. Instead, we'd just be importing the code we want to plug into the enhanced configure/build system, and building the commit history that shows how that new system was developed.